resources/eventing/charts/nats/templates/statefulset.yaml

```diff
@@ -99,6 +106,35 @@ spec:
             mountPath: /etc/nats-config
           - name: pid
             mountPath: /var/run/nats
+
+      ##############################
+      #                            #
+      #  NATS Prometheus Exporter  #
+      #                            #
+      ##############################
+      {{- if and .Values.exporter.enabled .Values.global.jetstream.enabled }}
+      - name: metrics
+        image: "{{ include "imageurl" (dict "reg" .Values.global.containerRegistry "img" .Values.global.images.prometheus_nats_exporter) }}"
+        imagePullPolicy: {{ .Values.exporter.pullPolicy }}
+        {{- if .Values.exporter.securityContext }}
+        securityContext:
+          {{- toYaml .Values.exporter.securityContext | nindent 10 }}
+        {{- end }}
+        resources:
+          {{- toYaml .Values.exporter.resources | nindent 10 }}
+        args:
+        - -connz
+        - -routez
+        - -subz
+        - -varz
+        - -prefix=nats
+        - -use_internal_server_id
+        - -jsz=all
+        - http://localhost:8222/
+        ports:
+          - containerPort: 7777
+            name: metrics
```

Comment:
The port naming in NATS sub-chart does not follow this naming convention, (https://github.com/kyma-project/kyma/blob/c2deb56effe24ac68f8bc990f82b2bf07c7bb43f/resources/eventing/charts/nats/templates/statefulset.yaml). So if we keep it like this then it would be consistent here.
IMO, this naming convention is important in k8s/svc because of Istio.